below/render/src/default_configs.rs

```diff
@@ -778,7 +778,32 @@ impl HasRenderConfig for model::SystemModel {
     }
 }
 
-impl HasRenderConfigForDump for model::SystemModel {}
+impl HasRenderConfigForDump for model::SystemModel {
+    fn get_openmetrics_config_for_dump(
+        &self,
+        field_id: &Self::FieldId,
+    ) -> Option<RenderOpenMetricsConfigBuilder> {
+        use model::SystemModelFieldId::*;
+        match field_id {
+            // We tag all metrics with the hostname already
+            Hostname => None,
+            // OpenMetrics does not support strings
+            KernelVersion => None,
+            // OpenMetrics does not support strings
+            OsRelease => None,
+            Stat(field_id) => self.stat.get_openmetrics_config_for_dump(field_id),
+            Cpu(field_id) => self.total_cpu.get_openmetrics_config_for_dump(field_id),
+            // `system` dump does not support individual CPU metrics
+            Cpus(_) => None,
```

Comment:
You can do the following:

`below dump system -f cpus.<N>.usage_pct --begin "5 min ago"`

Reply:
Ah I see. Fixed